src/api/ledger/accountinfo.js

```diff
@@ -18,21 +55,23 @@ function formatAccountInfo(response) {
   });
 }
 
-function getAccountInfoAsync(account, options, callback) {
-  validate.address(account);
-  validate.getAccountInfoOptions(options);
+function getAccountInfoAsync(account: string, options: AccountInfoOptions,
+  callback: AccountInfoCallback) {
+    validate.address(account);
+    validate.getAccountInfoOptions(options);
 
-  const request = {
-    account: account,
-    ledger: options.ledgerVersion || 'validated'
-  };
+    const request = {
```

Comment:
How to do it right?
First line of function 
`function getAccountInfoAsync(account: string, options: AccountInfoOptions,
  callback: AccountInfoCallback) {`
is too long, so I moved part of it to new line and indented it, but that requires to indent more all other function lines.